### PR TITLE
[flink] Remove Redundant Test Methods

### PR DESCRIPTION
**Pull request by MonsterChenzhuo**
### Purpose

Remove Redundant Test Methods

### CODE REVIEW

**Comment by MonsterChenzhuo** [comment]
@yuzelin PTAL,thanks.

File: paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/Action.java [hunk 1]
```diff
@@ -18,9 +18,13 @@
...
 
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
 /** Abstract class for Flink actions. */
 public interface Action {
 
     /** The execution method of the action. */
     void run() throws Exception;
+
+    default void build(StreamExecutionEnvironment env) throws Exception {}
```
Comment (yuzelin) [inline paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/Action.java:29]: Actions that override this method should be annotated.

Comment (MonsterChenzhuo) [reply paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/Action.java:42]: done.

**Review by yuzelin — APPROVED** [review]
+1

**Pull request merged**